### PR TITLE
Sheets Analysis

### PR DESCRIPTION
**Pull request by danieldortega77**
Separate module for analyzing a Google Sheet.

All changes to script.js and cover.css are the same as #15

oAuth.js is the same as #23

### CODE REVIEW

**Review by chrismohrii — APPROVED** [review]
LGTM - just some of my comments from the Docs PR also apply here

File: capstone/src/main/webapp/sheetsScript.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+ */
+async function getTextFromSheet(spreadsheetId) {
+  const response = await gapi.client.sheets.spreadsheets.values.get({
+    spreadsheetId: spreadsheetId,
+    range: 'Sheet1!A1:YY',
+  });
+  const range = await response.result;
+  if (range.values.length > 0) {
+    let text = '';
+    for (i = 0; i < range.values.length; i++) {
```
Comment (olu144) [inline capstone/src/main/webapp/sheetsScript.js]: nit: lets make this for (let i = 0;.....) instead of (i = 0;....), same goes for other loops

**Review by olu144 — APPROVED** [review]

**Comment by olu144** [comment]
LGTM !

File: capstone/src/main/webapp/sheetsScript.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+// Array of API discovery doc URLs for APIs
+const DISCOVERY_DOCS = ["https://sheets.googleapis.com/$discovery/rest?version=v4"];
+
+// Authorization scopes required by the API; multiple scopes can be
+// included, separated by spaces.
+const SCOPES = "https://www.googleapis.com/auth/spreadsheets";
+
+/** Collects the user's input and submits it for analysis */
+async function gatherSheetsInput() {
```
Comment (dborkan) [inline capstone/src/main/webapp/sheetsScript.js:26]: Just FYI, it seems like this `gatherSheetsInput` is very similar to `gatherDocsInput` from #23.  Possibly in a follow up CL you might want to refactor them so they use more similar code (possibly making some shared functions, or even making classes that inherit/implement the same interface)

**Review by dborkan — APPROVED** [review]
LGTM

File: capstone/src/main/webapp/sheetsScript.js (hunk 2 above)
Comment (danieldortega77) [reply capstone/src/main/webapp/sheetsScript.js:26]: Noted - I'll see if I can do this

**Review by dslucas — APPROVED** [review]

**Pull request merged**